<launch>
    <arg name="name" value="isd4000"/>
    <arg name="sensor" doc="Sensor name" value="isd4000"/>
    <node name="$(arg name)" pkg="ds_sensors" type="sensor" args="--sensor=$(arg sensor)" output="screen">
        <param name="latitude_deg" value="41.0007"/>
        <param name="health_check_period" type="double" value="1"/>
        <param name="message_timeout" type="double" value="5"/>
        <param name="descriptive_name" value="ISD4000 depth sensor"/>
        <param name="depth_unit" value="bar"/>
        <param name="frame_id" value="isd4000_link"/>
        <param name="instrument/type" value="SERIAL"/>
        <param name="instrument/port" value="/dev/ttyUSB0"/>
        <param name="instrument/baud" value="9600"/>
        <param name="instrument/parity" value="none"/>
        <param name="instrument/stop_bits" value="1"/>
        <param name="instrument/data_bits" value="8"/>
        <param name="instrument/matcher" value="match_char"/>
        <param name="instrument/delimiter" value="0A"/>
    </node>
</launch>